Gemfile

```diff
@@ -6,6 +6,9 @@ gem 'materialize-sass'
 
 gem "paperclip", "~> 4.3"
 
+gem 'globalize', '~> 5.0.0'
+gem 'rails-i18n', '~> 4.0.0'
```

Comment:
Did you need to add this gem in the Gemfile for any particular reason? Rails projects have support to internationalization without having to add the gem manually.